src/Sulu/Bundle/AdminBundle/Resources/public/js/components/csv-export/main.js

```diff
@@ -12,116 +12,15 @@
  * @class CSV-Export
  * @constructor
  */
-define(['jquery', 'underscore', 'text!./form.html'], function($, _, form) {
+define(['text!./form.html'], function(form) {
```

Comment:
a twig template here would make it easier to extend from the default template when we use some blocks to overwrite or add fields at the top or bottom else it need always to copy the whole form.html.